### ISSUE TITLE
AttributeError: 'tuple' object has no attribute 'dtype'

### ISSUE THREAD

**Issue by georgipeev** [ClickHouse/clickhouse-connect#108]
The above exception is thrown in any query that includes a `LowCardinality(String)` column. Regular `String` columns (including `Nullable(String)`) work fine. This problem does not happen with version 0.5.3.

Python version: 3.9
clickhouse_connect version: 0.5.4 

Stack trace:
```
Closed HTTP response due to unexpected exception
Traceback (most recent call last):
  File "/home/gpeev/test.py", line 8, in <module>
    df = client.query_df("select ticker from vendor_data limit 10")
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/driver/client.py", line 255, in query_df
    return self._context_query(locals(), use_numpy=True).df_result
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/driver/client.py", line 583, in _context_query
    return self._query_with_context((self.create_query_context(**kwargs)))
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/driver/httpclient.py", line 195, in _query_with_context
    query_result = self._transform.parse_response(byte_source, context)
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/driver/transform.py", line 50, in parse_response
    first_block = get_block()
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/driver/transform.py", line 42, in get_block
    column = col_type.read_column(source, num_rows, context)
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/datatypes/base.py", line 128, in read_column
    column = self.read_column_data(source, num_rows, ctx)
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/datatypes/base.py", line 140, in read_column_data
    return self._read_low_card_column(source, num_rows, ctx)
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/datatypes/base.py", line 223, in _read_low_card_column
    if ctx.use_numpy and not (self.nullable and use_none) and keys.dtype != np.object_:
AttributeError: 'tuple' object has no attribute 'dtype'
```

**Comment by genzgd** [comment]
Yes, this is broken.  I should have a fix out later today.

**Comment by genzgd** [comment]
v0.5.5 released with a fix.  Please validate if possible.

**Comment by georgipeev** [comment]
The issue seems to be fixed in 0.5.5. Thanks!

**Comment by genzgd** [comment]
Thanks for the report and for trying new versions.   Unfortunately I missed that test case but I hope you find the performance improvement for Pandas Dataframes was worth it.
